### DISCUSSION TITLE
CustomProduct building

### DISCUSSION THREAD

**Discussion by bcerripromethee** [sertit/eoreader#147] [Q&A] [answered]
Hey,

I am trying to use the `CustomProduct` feature to create my own product. I have a .zip folder which contains single bands, metadata (in a yaml format), a panchromatic band, the footprint, ... By exploring your feature, I've seen that EOReader needs an already coregistered product and we can't attached the metadata file.

Am i missing a part or maybe CustomProduct is not adapted to my needs ?

**Comment by remi-braun** [comment, marked as answer]
Hello,

The Custom Product has to be created like in the [notebook](https://eoreader.readthedocs.io/en/latest/notebooks/custom.html) (and the help is [here](https://eoreader.readthedocs.io/en/latest/custom.html))

Unfortunately, for now it doesn't handle separate bands, mtd file, and I don't know if it works with rasters in zip 🤔 
Sorry for that. Maybe if this is really important you could try to create a draft of the feature?
